atom/renderer/api/lib/desktop-capturer.coffee

```diff
@@ -0,0 +1,10 @@
+ipc = require 'ipc'
+NativeImage = require 'native-image'
+
+getSources = (options, callback) ->
+  ipc.send 'ATOM_BROWSER_DESKTOP_CAPTURER_GET_SOURCES', options
+  ipc.once 'ATOM_RENDERER_DESKTOP_CAPTURER_RESULT', (sources) ->
```

Comment:
Ah I mistook the code, actually problem only happens when you call `getSources` with different options:

```coffee
dc.getSources {types:['window']}, -> console.log(arguments)
dc.getSources {types:['window', 'screen']}, -> console.log(arguments)
```

the result of first `getSources` will also be returned for the second `getSources`.